charts/k6-operator/templates/service.yaml

```diff
@@ -17,5 +17,5 @@ spec:
     port: 8443
     targetPort: https
   selector:
-    control-plane: "controller-manager"
+    {{- include "k6-operator.selectorLabels" . | nindent 4 }}
```

Comment:
I suggest to fix the Chart so that `control-plane: "controller-manager"` would work. Rationale: it'd be preferable to keep the labeling consistent with kustomize base, i.e. here:
https://github.com/grafana/k6-operator/blob/main/config/manager/manager.yaml

Reply:
Sure, whatever changes you guys want. I have moved the label inside the deployment template instead.